content/en/blog/_posts/2024-05-09-gateway-api-v1.1/index.md

```diff
@@ -0,0 +1,258 @@
+---
+layout: blog
+title: "Gateway API v1.1: Service mesh, GRPCRoute, and a whole lot more"
+date: 2024-05-09T09:00:00-08:00
+slug: gateway-api-v1-1
+author: >
+  [Richard Belleville](https://github.com/gnossen) (Google),
+  [Frank Budinsky](https://github.com/frankbu) (IBM),
+  [Arko Dasgupta](https://github.com/arkodg) (Tetrate),
+  [Flynn](https://github.com/kflynn) (Buoyant),
+  [Candace Holman](https://github.com/candita) (Red Hat),
+  [John Howard](https://github.com/howardjohn) (Solo.io),
+  [Christine Kim](https://github.com/xtineskim) (Isovalent),
+  [Mattia Lavacca](https://github.com/mlavacca) (Kong),
+  [Keith Mattix](https://github.com/keithmattix) (Microsoft),
+  [Mike Morris](https://github.com/mikemorris) (Microsoft),
+  [Rob Scott](https://github.com/robscott) (Google),
+  [Grant Spence](https://github.com/gcs278) (Red Hat),
+  [Shane Utt](https://github.com/shaneutt) (Kong),
+  [Gina Yeh](https://github.com/ginayeh) (Google),
+  and other review and release note contributors
```

Comment:
I'm still not keen on this, per https://github.com/kubernetes/website/pull/45971#discussion_r1584271415

Reply:
Here's how that renders on my screen:
![Screenshot](https://github.com/kubernetes/website/assets/22591623/b6d8f311-1bc2-4229-9853-6a600a42f78d)

Reply:
It's good to have a small team formally author on the release announcement, even if a wider team have helped contribute to making the release notes and announcement happen.

Reply:
@sftim that's entirely fair, do you have any recommendations for a max number of authors we should accept? Even if we were to say max(1) per company for example, we'd still end up with ~9 given the current list. Would it be reasonable to have authors of individual sections listed by the section header?

Reply:
One, two or three authors works for a blog article. Any more than that and it gets complex (or we can designate it as a team effort, as we do with Kubernetes releases). Even then, release comms has a lead and a small number of helpers.

> Would it be reasonable to have authors of individual sections listed by the section header?

We don't usually do credit like this, so my (very gentle) advice is not to do it. Handling credit for open source contributions is best kept separate from blogs and release announcements. Again, you can hyperlink to a credits page.